setup-runner-windows.ps1

```diff
@@ -42,7 +42,7 @@ if (!$HasTestSigning) {
 # Enable PowerShell remoting to peer.
 Write-Host "Enabling Remote PowerShell."
 "$PeerIp netperf-peer" | Out-File -Encoding ASCII -Append "$env:SystemRoot\System32\drivers\etc\hosts"
-pwsh -Command 'Enable-PSRemoting -Force'
+Enable-PSRemoting -Force
```

Comment:
I had to do this when I was setting up the Azure machines. Because when you do this, it only allows normal PowerShell to make a connection, not pwsh (the new powershell).

Reply:
Interesting, cause I get thrown an error cause "pwsh" is not recognized... did u have this issue too?